crawl-ref/docs/develop/levels/syntax.txt

```diff
@@ -496,18 +491,17 @@ ITEM:     (list of items, separated by comma)
           possibility is
           [possibility's weight: / sum of all weight:s in that array position]
 
-          For example, the following line makes letter "d" into a bread ration
-          with 50% chance, or apple or orange with 25% chance each:
+          For example, the following line makes letter "d" into a cloak with
+          50% chance, or hat or helmet with 25% chance each:
 
-            ITEM: bread ration / w:5 apple / w:5 orange
+            ITEM: cloak / w:5 hat / w:5 helmet
 
           Modifiers:
           * "q:N" sets the item quantity to N (if N > 0). Does nothing
              if the item is not stackable.
           * "charges:N" sets the number of charges for wands.
-          * "plus:N" and "plus2:N" set the item pluses for weapons, armour,
-            jewellery (such as rings of intelligence or slaying), and
-            throwing nets.
+          * "plus:N" sets the item plus for weapons, armour, and jewellery
+            (such as rings of intelligence or slaying).
```

Comment:
Jewellery has fixed pluses now, but maybe vaults can still override that? Haven't checked carefully.

Reply:
It's still possible to place custom jewellery in a vault. Stat rings and rings of evasion/protection work fine with ``plus:100``. Also, meatsprint has a lot of custom rings of slaying.